scripts/combine-reports.gmp.py

```diff
@@ -104,7 +104,7 @@ def send_report(gmp: Gmp, args: Namespace, combined_report: e.Element) -> str:
 
     combined_report = e.tostring(combined_report)
 
-    res = gmp.import_report(combined_report, task_id=task_id)
+    res = gmp.import_report(combined_report, task_id=task_id, in_assets=True)
```

Comment:
Does this change really belong to this PR?

Reply:
No ... not really ... I can revert it, and make another PR for it ...

Reply:
Would be nice to do that